.github/workflows/pre-commit.yml

```diff
@@ -24,14 +24,37 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
-  check-chart-version-bump:
+  chart-version-bumps:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - name: Check if Chart.yaml versions are bumped if relevant
+      - name: Setup updatecli
+        uses: updatecli/updatecli-action@v2
+
+      - name: Bump any charts that require it
         run: |
-          make validate-chart-version
+          make updatecli/internal
+          git diff > diff.txt
```

Comment:
too much logic, why can't this be folded into the makefile?

Reply:
Ideally the github action would do inline code comments that the author could accept to update their PR seamlessly.

In the abasence of being able to do that I thought a comment on the PR was the next best ...